packages/flutter_tools/lib/src/base/user_messages.dart

```diff
@@ -292,4 +294,16 @@ class UserMessages {
       'In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.\n'
       'Read more about iOS versioning at\n'
       'https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html\n';
+
+  String _androidSdkInstallUrl(Platform platform) {
+    if (platform.isMacOS) {
+      return 'https://flutter.dev/docs/get-started/install/macos#android-setup';
```

Comment:
Maybe pull 'https://flutter.dev/docs/get-started/install' and 'android-setup' out into locals to avoid repeating them.

Reply:
Done, though I think it's harder to read.
(The more correct thing would be to construct a Uri but that seems over-engineered for this one.)